raiden/tests/unit/test_pfs_integration.py

```diff
@@ -433,7 +443,8 @@ def test_routing_mocked_pfs_unavailable_peer(chain_state, token_network_state, o
                 ],
                 "fees": 0,
             }
-        ]
+        ],
+        "feedback_token": "381e4a005a4d4687ac200fa1acd15c6f",
```

Comment:
Nitpick: Why not write this as `DEFAULT_FEEDBACK_TOKEN.hex` here and below, and assert for the matching value instead of just `is not None`?